### ISSUE TITLE
Reg. second step

### ISSUE THREAD

**Issue by RPIASRI** [canzarlab/McSplicer#5]
I want to use McSplicer for predicting AS events in my transcriptome dataset. I had taken annotation in gtf format  which is generated by stringtie and run the first step and annotation_refined.gtf file  is created.

Further I took the bam files generated through samtools sort and index it to create bai file

but when I am running second step of the program ( ./bin/sigcount <alignments.bam> <annotation_refined.gtf> <outfile-prefix>), I am getting following error:
"Work with given annotation
WARNING: Mate could not be found for:"

Kindly help me with the same.

**Comment by canzarlab** [comment]
This seems to be a warning, not an error. Did you get (non-zero) counts in the .cnt files? Did you provide single or paired-end reads?

**Comment by RPIASRI** [comment]
No, I didnt get any count in .cnt file. It was totally blank. I provide paired-end reads. Kindly help me

**Comment by canzarlab** [comment]
I have just added a new binary for running sigcount with paired-end reads: sigcount_pe in folder bin. Please try and let me know if this solved your problems.

**Comment by RPIASRI** [comment]
I tried this option but is getting same error, whether using sigcount or sigcount_pe
Infact, i reinstall it and then executed but the error was there and zero counts in .cnt file.
For your reference, I am providing initial few lines of the error, here:

Work with given annotation
WARNING: Mate could not be found for:
SRR12772292.9157794	369	CM000126.1	195323	1	46M	=	195261	0	ACACACTGCGTCTTGGAGTCCGTGCTACAGCTGGCTACAAATCTGT	:::,::,FF,,:FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF	AS:i:0	ZS:i:0	XN:i:0	XM:i:0	XO:i:0	XG:i:0	NM:i:0	MD:Z:46	YT:Z:UP	NH:i:5
WARNING: Mate could not be found for:
SRR12772292.3462499	385	CM000126.1	244808	1	53M1105N4M	=	244728	0	CAAGAACTGTGTTCTCAACTTGGAGCTGTGTTAGGAATGCTTGGGGACTGCTGTGGG	FFFF:FFFFFFFFFFFFFFFF:F:FFFFFFFFFFF,FFFFFFFFFF,FFF,,FF,FF	AS:i:-3	ZS:i:-3	XN:i:0	XM:i:1	XO:i:0	XG:i:0	NM:i:1	MD:Z:54C2	YT:Z:UP	XS:A:+	NH:i:2

**Comment by canzarlab** [comment]
sorry for these problem. would you be able to share your bam file or a random sample (e.g. 10% of reads) of it?

**Comment by alig99** [comment]
Hi,
Just wondering if this issue was resolved as I have the same issue receiving segmentation fault (core dumped)
when running sigcount_pe with paired end reads.   Thank you ali

Work with given annotation
WARNING: Mate could not be found for:
R100400180029:20230620153811:V350160349:2:1087843:2:51:AATACCGT	177	X	111124287	60	148M	17	50185629	0	TGAAGATAAACTGAAGAATAATGAACCACATTCCATGCAAGAGATTTGTGACTATGCAAGAGGAATTTCCAGAGACATGTCTTTAGACACTGAAATGAATCCAGAAAAAAATTTGTTGGGAGTAAGGGATGGTTTGAGGAATTTAGGG	EGB5CA*BBCFCGCDED;E7=/,B:EGB<BFCE4DDG8@@GBD>C?F>DD>>E'EB?BD@C?1A1&EE)EB+D9DE?>?;$9F8C=@G<HCCBBDE8?C:DAC<>BB8BADBE=B<?G;CAGDDBG>E?EC<CA?>B7$+ADAE<F@0	AS:i:-2	XN:i:0	XM:i:1	XO:i:0	XG:i:0	YT:Z:UP	XS:A:+	NH:i:1	MD:Z:68T79	NM:i:1
WARNING: Mate could not be found for:
R100400180029:20230620153811:V350160349:4:247001:1:11:GCGAGATC	97	X	111192586	60	33M1682N115M	22	45323385	0	TAAGAACCCTAATATTGTTAATTATTTAGATAGCTACTTGGTGGGTGATGAACTATGGGTAGTCATGGAATACTTGGCTGGTGGCTCTCTGACTGATGTGGTCACAGAGACCTGTATGGATGAAGGACAGATAGCAGCTGTCTGCAGA	DF0GDDG?C3FD9E@=1ABEE>3@>CCDE9CD+HCEFBCCFCF951GCAGBEGCFB>?FAFEDCDCDBEEDDGD=FEFD:EBCFF=GCFDFCCCF5A1CGFDFEDE>CG.FFCCC9ACEECH*?1B@C<GD4EFGB,EB?D8B>G)EE	AS:i:0	XN:i:0	XM:i:0	XO:i:0	XG:i:0	YT:Z:UP	XS:A:+	NH:i:1	MD:Z:148	NM:i:0

**Comment by israa-alqassem** [comment]
Hi,
Sorry for the late response.
Do you have paired single end reads? Could you share your bam file with me so I can look into this error?


> Hi, Just wondering if this issue was resolved as I have the same issue receiving segmentation fault (core dumped) when running sigcount_pe with paired end reads. Thank you ali
> 
> Work with given annotation WARNING: Mate could not be found for: R100400180029:20230620153811:V350160349:2:1087843:2:51:AATACCGT 177 X 111124287 60 148M 17 50185629 0 TGAAGATAAACTGAAGAATAATGAACCACATTCCATGCAAGAGATTTGTGACTATGCAAGAGGAATTTCCAGAGACATGTCTTTAGACACTGAAATGAATCCAGAAAAAAATTTGTTGGGAGTAAGGGATGGTTTGAGGAATTTAGGG EGB5CA_BBCFCGCDED;E7=/,B:EGB<BFCE4DDG8@@gbd>C?F>DD>>E'EB?BD@C?1A1&EE)EB+D9DE?>?;$9F8C=@g<HCCBBDE8?C:DAC<>BB8BADBE=BE?ECB7$+ADAE<F@0 AS:i:-2 XN:i:0 XM:i:1 XO:i:0 XG:i:0 YT:Z:UP XS:A:+ NH:i:1 MD:Z:68T79 NM:i:1 WARNING: Mate could not be found for: R100400180029:20230620153811:V350160349:4:247001:1:11:GCGAGATC 97 X 111192586 60 33M1682N115M 22 45323385 0 TAAGAACCCTAATATTGTTAATTATTTAGATAGCTACTTGGTGGGTGATGAACTATGGGTAGTCATGGAATACTTGGCTGGTGGCTCTCTGACTGATGTGGTCACAGAGACCTGTATGGATGAAGGACAGATAGCAGCTGTCTGCAGA DF0GDDG?C3FD9E@=1ABEE>3@>CCDE9CD+HCEFBCCFCF951GCAGBEGCFB>?FAFEDCDCDBEEDDGD=FEFD:EBCFF=GCFDFCCCF5A1CGFDFEDE>CG.FFCCC9ACEECH_?1B@C<GD4EFGB,EB?D8B>G)EE AS:i:0 XN:i:0 XM:i:0 XO:i:0 XG:i:0 YT:Z:UP XS:A:+ NH:i:1 MD:Z:148 NM:i:0
